### PR TITLE
Des/feature/panels gebruikersbeheer

### PR DESCRIPTION
**Pull request by evajacobs**
Old:
Screenshot 2020-06-19 at 16 07 27

New: 
Screenshot 2020-06-19 at 16 02 30

This PR:
- Add key-value component
- Use Panel component on user settings detail page

### CODE REVIEW

File: app/components/key-value/component.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+
+export default class KeyValueComponent extends Component {
+  get layout() {
+    if (this.args.layout) {
+      return "vlc-key-value-item--"+this.args.layout;
```
Comment (rafael-mindreau) [inline app/components/key-value/component.js]: 😬  spacing for operator
```javascript
return "vlc-key-value-item--" + this.args.layout
```

File: app/components/key-value/component.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    }else{
```
Comment (rafael-mindreau) [inline app/components/key-value/component.js]:
```javascript
} else {
```

**Review by rafael-mindreau — CHANGES_REQUESTED** [review]
😬  Some code style mishaps, only small stuff. All the rest looks great!

**Review by rafael-mindreau — APPROVED** [review]
🔝

**Review by woutersf — APPROVED** [review]

**Pull request merged**